src/controllers/index.js

```diff
@@ -3,17 +3,26 @@ const router = express.Router();
 const insertProject = require("./insertProject");
 const home = require("./home");
 const signup = require("./signup");
+const login = require('./login');
+const project = require('./singleProject');
+const checkuser = require('./checkUser')
 
 router.get("/", home.get);
-router.post("/", home.post);
+// router.post("/", home.post);
```

Comment:
what do you use `post` method for??
and what could you use it for from the homepage?